docs/content/installation/building-ingress-controller-image.md

```diff
@@ -1,74 +1,86 @@
 ---
-title: Building the Ingress Controller Image
-description: "This document explains how to build an Ingress Controller image using the source code."
+title: Building NGINX Ingress Controller Image
+description: "This document explains how to build an NGINX Ingress Controller image using the source code."
 weight: 2200
 doctypes: [""]
 toc: true
 docs: "DOCS-601"
```

Comment:
```suggestion
```
Doing some due diligence to axe this since there'll be a new system in the future.